src/bundle/Resources/views/admin/systeminfo/database.html.twig

```diff
@@ -2,14 +2,34 @@
 
 <!-- Tab name: {{ 'tab.name.database'|trans|desc('Database') }} -->
 
-<h1>{{ 'database'|trans|desc('Database') }}</h1>
-<dl>
-    <dt>{{ 'database.type'|trans|desc('Type') }}</dt>
-    <dd>{{ info.type }}</dd>
-    <dt>{{ 'database.name'|trans|desc('Name') }}</dt>
-    <dd>{{ info.name }}</dd>
-    <dt>{{ 'database.host'|trans|desc('Host') }}</dt>
-    <dd>{{ info.host }}</dd>
-    <dt>{{ 'database.username'|trans|desc('Username') }}</dt>
-    <dd>{{ info.username }}</dd>
-</dl>
+<div class="flex-wrapper">
+    <div class="ez-contentbox ez-contentbox-list">
```

Comment:
maybe this could be a `<table>`?

Reply:
I did it based on https://ezsystems.github.io/ui-guidelines/#components-contentbox, and there it is not a table. Do you think it is necessary to change this?

Reply:
I think yes. When I open guidelines in my FF the contentbox is misaligned, I think a table would never let to happen this.